### PR TITLE
[Community]: Added classification_location parameter in PebbloSafeLoader.

### PR DESCRIPTION
**Pull request by rahul-trip**
Description: Add classifier_location feature flag. This flag enables Pebblo to decide the classifier location, local or pebblo-cloud.
Unit Tests: N/A
Documentation: N/A

### CODE REVIEW

**Comment by rahul-trip** [comment]
Hi @baskaryan 
Kindly review. Thanks!

**Comment by rahul-trip** [comment]
Hi @eyurtsev, @hwchase17
Kindly review and merge if looks fine.

File: libs/community/langchain_community/chains/pebblo_retrieval/base.py [hunk 1]
```diff
@@ -321,38 +333,44 @@ def _get_app_details(app_name, owner, description, llm, **kwargs) -> App:  # typ
         return app
 
     @staticmethod
-    def _send_discover(app, api_key, classifier_url) -> None:  # type: ignore
+    def _send_discover(app, api_key, classifier_url, classifier_location) -> None:  # type: ignore
```
Comment (eyurtsev) [inline libs/community/langchain_community/chains/pebblo_retrieval/base.py:336]: type annotations missing here

File: libs/community/langchain_community/utilities/pebblo.py [hunk 2]
```diff
@@ -150,6 +150,7 @@ class Doc(BaseModel):
     loader_details: dict
     loading_end: bool
     source_owner: str
+    classifier_location: str
```
Comment (eyurtsev) [inline libs/community/langchain_community/utilities/pebblo.py:153]: * Could you update the doc-string?
* The doc-strings should be below the attribute not in the model doc-string

**Comment by eyurtsev** [comment]
@Raj725 are you able to review?

**Comment by eyurtsev** [comment]
@rahul-trip if you're working at daxa-ai, merge time can be significantly reduced by getting another daxa-ai engineer to review the changes

File: libs/community/langchain_community/document_loaders/pebblo.py [hunk 3]
```diff
@@ -46,6 +46,7 @@ def __init__(
         api_key: Optional[str] = None,
         load_semantic: bool = False,
         classifier_url: Optional[str] = None,
+        classifier_location: str = "local",
```
Comment (eyurtsev) [inline libs/community/langchain_community/document_loaders/pebblo.py:49]: Use `*` prior to new named arguments

File: libs/community/langchain_community/chains/pebblo_retrieval/base.py (hunk 1 above)
Comment (rahul-trip) [reply libs/community/langchain_community/chains/pebblo_retrieval/base.py:336]: Added missing type annotations throughout file.

File: libs/community/langchain_community/utilities/pebblo.py (hunk 2 above)
Comment (rahul-trip) [reply libs/community/langchain_community/utilities/pebblo.py:153]: Updated all the models with suggested doc-string style.

File: libs/community/langchain_community/document_loaders/pebblo.py (hunk 3 above)
Comment (rahul-trip) [reply libs/community/langchain_community/document_loaders/pebblo.py:49]: Added as suggested. 

@eyurtsev , Can you please help me understanding how is this going to help please. Is this a new norm that I am unaware of, or is it langchain coding style, please.

**Comment by rahul-trip** [comment]
> @rahul-trip if you're working at daxa-ai, merge time can be significantly reduced by getting another daxa-ai engineer to review the changes

@eyurtsev we do review each PR internally and post the branch here once internal review is approved (we do this in our fork.). From now, we will start the approval process here as suggested.

@Raj725, please check if you are able to approve.

**Comment by Raj725** [comment]
> @Raj725 are you able to review?

@eyurtsev  Apologies for the delay. I've reviewed it, and it looks good to me.

**Review by Raj725 — APPROVED** [review]
LGTM

**Comment by eyurtsev** [comment]
@rahul-trip  / @Raj725 thanks! Your existing process is great. Mainly helpful to see a final sign off from another person on the team!

**Pull request merged**